### PR TITLE
Add support for Surface Laptop 2 & 3

### PR DESCRIPTION
**Pull request by dorianclay**
Added touchpad measurements for Surface Laptop 2 13" and 3 15". Tested on hardware for both.

### CODE REVIEW

File: hwdb.d/60-evdev.hwdb [hunk 1]
```diff
@@ -664,6 +664,24 @@ evdev:name:ALPS0001:00 0911:5288 Touchpad:dmi:*svnStarLabs:pnLite:*
  EVDEV_ABS_01=51:950:15
  EVDEV_ABS_35=55:1750:16
  EVDEV_ABS_36=51:950:15
+ 
+#########################################
+# Surface
```
Comment (yuwata) [inline hwdb.d/60-evdev.hwdb]: Microsoft??

**Review by yuwata — CHANGES_REQUESTED** [review]
LGTM, except for the section title in the comment.

File: hwdb.d/60-evdev.hwdb (hunk 1 above)
Comment (dorianclay) [reply hwdb.d/60-evdev.hwdb]: oops

File: hwdb.d/60-evdev.hwdb [hunk 2]
```diff
@@ -664,7 +682,7 @@ evdev:name:ALPS0001:00 0911:5288 Touchpad:dmi:*svnStarLabs:pnLite:*
  EVDEV_ABS_01=51:950:15
  EVDEV_ABS_35=55:1750:16
  EVDEV_ABS_36=51:950:15
-
+
```
Comment (bluca) [inline hwdb.d/60-evdev.hwdb]: Spurious whitespace

Comment (dorianclay) [reply hwdb.d/60-evdev.hwdb]: an eye for the invisible...

**Pull request merged**